build.gradle.kts

```diff
@@ -46,8 +46,8 @@ dependencies {
   testImplementation("org.springframework.security:spring-security-test")
   testImplementation("com.squareup.okhttp3:mockwebserver:4.9.2")
   testImplementation("com.squareup.okhttp3:okhttp:4.9.2")
+  testImplementation("com.h2database:h2")
```

Comment:
This was failing on a security alert also.  However it is only needed during the phase ergo this solves the alert from being thrown.